djstripe/models.py

```diff
@@ -541,20 +542,94 @@ def _attach_objects_hook(self, cls, data):
         if subscription:
             self.subscription = subscription
 
+        self._attach_invoice_items(cls, data)
+
+    def _attach_invoice_items(self, cls, data):
+        if not self.pk:
+            # InvoiceItems need a saved invoice because they're associated via
+            # a RelatedManager.  It might be better to make this pattern more
+            # generic with some sort-of _attach_objects_post_save_hook().
```

Comment:
I agree with this. Would you mind adding that post save hook? (Another PR is fine if that would cause any conflicts in the PR chain)